perftools/stresstesting/login-and-learn.yml

```diff
@@ -0,0 +1,22 @@
+config:
```

Comment:
Btw you can create timed jobs in Travis, so you can have a build automatically trigger once in a while and report back about benchmarking stuff.. this might be really cool to have in a separate repo...